app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    current_file = command.split()[1]
+    new_file = command.split()[2]
```

Comment:
don't split your command twice
create a separated variable for command.split() and use it on demand